src/uct/api/uct.h

```diff
@@ -295,6 +299,8 @@ struct uct_iface_attr {
             size_t           max_zcopy;  /**< Total max. size (incl. the header
                                               and total of @ref uct_iov_t::length
                                               of the @a iov parameter) */
+            uint16_t         opt_zcopy_align; /**< Alignment parameter */
```

Comment:
let's make it size_t - uint16_t does not really save memory (due to padding) and size_t is more generic